scss/_dropdown.scss

```diff
@@ -105,7 +105,7 @@
 }
 
 // When enabled Popper.js, reset basic dropdown position
-// stylelint-disable no-duplicate-selectors
+// stylelint-disable-next-line no-duplicate-selectors
```

Comment:
I was hoping we could reduce the amount of duplicate selectors when possible later :)